cache/README.md

```diff
@@ -0,0 +1,59 @@
+# Cache
+
+## **Requests**
+
+|          Route           |                       Description                       |
+| :----------------------: | :-----------------------------------------------------: |
+| [GET /cache](#get-cache) | Returns the cloudflare cache meta data of last 24 hours |

```

Comment:
What are your thoughts on calling it something like `cloudflareCache` or `cfcache` or `cachecf` or `cache/cf`

Reply:
I think `cfcache` is a better name but according to conventions we should nouns and which are clear and intuitive. Most people will confuse with `cf`